umap/static/umap/js/umap.features.js

```diff
@@ -36,6 +45,9 @@ U.FeatureMixin = {
     this.addInteractions()
     this.parentClass.prototype.initialize.call(this, latlng, options)
   },
+  _checkId: function (string) {
+    return typeof string !== 'undefined'
```

Comment:
Shouldn't we check that the id is in our internal format and not something else ? Or do we want to consume any non null string ? So people importing in uMap could control their ids ?

Reply:
You're correct. I've added a check to ensure ids are matching the expected format.